MiddlesexMA/src/main/java/com/openlattice/middlesex/MiddlesexBookings1.java

```diff
@@ -57,20 +57,22 @@
             .getLogger( MiddlesexBookings1.class );
     private static final RetrofitFactory.Environment environment = Environment.PRODUCTION;
     private static final DateTimeHelper              dtHelper    = new DateTimeHelper( DateTimeZone
+
             .forOffsetHours( -5 ), "MM/dd/yy" );
     private static final DateTimeHelper              bdHelper    = new DateTimeHelper( DateTimeZone
```

Comment:
if both dates are formatted the same, you don't need 2 DateTimeHelpers -- you can use the same one for all dates